js/Utils.js

```diff
@@ -115,3 +115,18 @@ function capitalize(s){
 function removeTrailingZeros(value) {
     return (value + '').replace(/(\.\d*?)0+$/, '$1').replace(/\.$/, '');
 }
+
+function parseDateStringOrRestoreHeightAsInteger(value) {
+    // Parse date string or restore height as integer
+    var _restoreHeight = 0;
```

Comment:
Also while we are at it, you can remove the `_` prefix here, it does not match our other coding style.